### ISSUE TITLE
Enable users to mark episodes as played

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-rn#461]

**Comment by mitchdowney** [comment]
This is blocked by https://github.com/react-native-kit/react-native-track-player/issues/472

Until we can automatically detect the end of episodes, the "mark played episodes" feature will be incomplete from a UX perspective.

**Comment by mitchdowney** [comment]
Duplicate of https://github.com/podverse/podverse-rn/issues/539

**Comment by mitchdowney** [comment]
Added a while ago :)
